bin/build_release

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -e
+
+if [ $# != 1 ]; then
+    echo $0 RELEASE_VERSION
+    false
+fi
+
+VERSION=$1
+shift
+
+ROOT=$(realpath $(dirname $0)/..)
+cd $ROOT
+
+changes=`git status --porcelain`
+if [ -n "$changes" ]; then
+    echo Working tree not clean.
+    false
+fi
+
+git checkout master
+
+changed=`git diff --name-only release_stable docs/changelog.md`
+if [ -z "$changed" ]; then
+    git log release_stable..HEAD --pretty=oneline | sed -e 's/[a-z0-9]+/\*/g'
+    echo docs/changelog.md has not been updated since last release_stable
+    echo Use the log lines above for inspiration.
+    false
+fi
+
+tagged=`git rev-list -n 1 $VERSION 2>/dev/null` || true
+if [ -n "$tagged" ]; then
+    echo Tag $VERSION already exists. Try the next version.
+    false
+fi
+
+source etc/config_base.sh
+
+if [ "$host_tests" != config/modules/all.conf ]; then
+    echo Configure your system with host_tests=config/modules/all.conf
+    false
+fi
+
+cmd/build force $VERSION
+
+cmd/build push
+
+cat > /tmp/git_expected.txt <<EOF
+ M etc/docker_images.txt
+ M etc/docker_images.ver
+EOF
+
+git status --porcelain > /tmp/git_status.txt
+
+if ! diff /tmp/git_expected.txt /tmp/git_status.txt; then
+    echo Expected build images not found. Something went wrong.
+    false
+fi
+
+git commit -a -m "$VERSION release"
+git tag -a $VERSION -m "$VERSION release"
+git push
+git push --tags
+
+faucetgit=`git config remote.faucet.url`
```

Comment:
This would be applicable if the users fork of the repo is set up as a remote called 'faucet'? If so, maybe add a comment to that affect?